extensions/blocks/dialogue/dialogue.php

```diff
@@ -0,0 +1,245 @@
+<?php
+/**
+ * Dialogue Block.
+ *
+ * @since 9.3.0
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Dialogue;
+
+use Automattic\Jetpack\Blocks;
+use Jetpack_Gutenberg;
+
+const FEATURE_NAME = 'dialogue';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	$deprecated = function_exists( 'gutenberg_get_post_from_context' );
+	$uses       = $deprecated ? 'context' : 'uses_context';
+	Blocks::jetpack_register_block(
+		BLOCK_NAME,
+		array(
+			'render_callback' => __NAMESPACE__ . '\render_block',
+			$uses             => array(
+				'jetpack/conversation-participants',
+				'jetpack/conversation-showTimestamps',
+			),
+		)
+	);
+}
+add_action( 'init', __NAMESPACE__ . '\register_block' );
+
+/**
+ * Helper function to filter dialogue content,
+ * in order to provide a safe markup.
+ *
+ * @param string $content Dialogue content.
+ * @return string Safe dialgue content markup.
+ */
+function filter_content( $content ) {
+	if ( empty( $content ) ) {
+		return '';
+	}
+
+	return wp_kses(
```

Comment:
What would you think about using `wp_kses_post` here?

Reply:
sorry I missed your message. Addressing now.

Reply:
https://github.com/Automattic/jetpack/pull/18430